__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => {
+    return seed(testData);
+  });
+  
+afterAll(() => {
+    db.end();
+  });
+
+describe("NC News App", () => {
+    describe("GET /api/topics", () => {
+        test("Status: 200 when a successful request is made", () => {
+            return request(app)
+            .get("/api/topics")
+            .expect(200);
+        });
+
+        test("Status: 200 and replies with a JSON object of topics", () => {
```

Comment:
These two tests could be combined into one as we are tested the expected status code in both